rust-code-analysis-cli/src/main.rs

```diff
@@ -25,8 +22,24 @@ use std::{process, thread};
 use walkdir::{DirEntry, WalkDir};
 
 use formats::Format;
-use rust_code_analysis::*;
-use web::server;
+
+// Functions
+use rust_code_analysis::{
+    action, fix_includes, get_from_ext, get_function_spaces, guess_language, preprocess, read_file,
+    read_file_with_eol, write_file,
+};
+
+// Enums
+use rust_code_analysis::LANG;
+
+// Structs
+use rust_code_analysis::{
+    CommentRm, CommentRmCfg, Count, CountCfg, Dump, DumpCfg, Find, FindCfg, Function, FunctionCfg,
+    Metrics, MetricsCfg, PreprocParser, PreprocResults,
+};
+
+// Traits
+use rust_code_analysis::ParserTrait;
```

Comment:
Why splitting them? Let's move them all together under one `use`

Reply:
Actually, let's keep the old code here, the change isn't related to this PR

Reply:
I have problems with `*` sometimes, but you are right, I can open another PR